src/e2e/resources/simple-temple-expected/simple-temple-test-group/dao/dao.go

```diff
@@ -108,7 +108,7 @@ func (dao *DAO) ReadSimpleTempleTestGroup(input ReadSimpleTempleTestGroupInput)
 
 // UpdateSimpleTempleTestGroup updates the simpleTempleTestGroup in the datastore for a given ID, returning the newly updated simpleTempleTestGroup
 func (dao *DAO) UpdateSimpleTempleTestGroup(input UpdateSimpleTempleTestGroupInput) (*SimpleTempleTestGroup, error) {
-	row := executeQueryWithRowResponse(dao.DB, "UPDATE simple_temple_test_group SET WHERE id = $1;", input.ID)
+	row := executeQueryWithRowResponse(dao.DB, "UPDATE simple_temple_test_group SET WHERE id = $1 RETURNING id;", input.ID)
 
 	var simpleTempleTestGroup SimpleTempleTestGroup
 	err := row.Scan(&simpleTempleTestGroup.ID, &simpleTempleTestGroup.CreatedBy)
```

Comment:
This also looks a little broken (I know it wasn't your change - but should we be storing that too?)

@smithwjv is this on your radar?

Reply:
Yeah aha, made a card "Handle the case of no user-defined/client-facing attributes"